rslib/backend.proto

```diff
@@ -50,6 +50,10 @@ message NoteID {
   int64 nid = 1;
 }
 
+message NoteIDs {
```

Comment:
Looks like this is not needed then? Same for rsbackend.py.

Reply:
Yep, sorry, missed that.